### DISCUSSION TITLE
Admin created identities and is there a use for multiple schemas?

### DISCUSSION THREAD

**Discussion by Codebreaker101** [ory/kratos#2306] [Q&A] [answered]
I have an application where an administrator (kratos authenticated user, default schema) creates identities (username) and their credentials (password) through Web UI as well as CLI . There identities are not people but vehicles and they don't have an email address. Each vehicle will have an app on the dash where once the app is authenticated, it will have an infinite session length.

I am wondering what is the best way to achieve this, if it is even possible with kratos. Here what I have so far:
1. Create a new schema for the vehicles
2. Create identity using `/POST /identites` endpoint from a backend service 

Now that that is done, a few questions:

1. How to set credentials, in this case a password, for the vehicle? (all of the flows are using default identity schema)
2. How to authenticate a vehicle? (all of the flows are using default identity schema)
3. How to change session cookie validity duration per schema? 

Additionally, browsing trough the API docs, I don't see any usefulness in having multiple schemas since only one could ever be used for authentication, registration etc. Am I missing something?

**Comment by vinckr** [comment, marked as answer]
Hello @Codebreaker101 

Thanks for reaching out,
1. See this PR  https://github.com/ory/kratos/pull/2256 - This will be released properly soon 🎉 
2. I believe this issue addresses this: https://github.com/ory/kratos/issues/765, please let me know if not.
3. I think this is currently configured in client, so you could spin up multiple Kratos instances with different configuration, pointing to the same backend/frontend. Maybe there is another way I am missing now.
